test/gtest/uct/uct_test.cc

```diff
@@ -967,7 +968,13 @@ void uct_test::entity::mem_type_reg(uct_allocated_memory_t *mem,
                                     unsigned mem_flags) const
 {
     if (md_attr().reg_mem_types & UCS_BIT(mem->mem_type)) {
-        ucs_status_t status = uct_md_mem_reg(m_md, mem->address, mem->length,
+        /* Register memory respecting MD reg_alignment */
+        size_t reg_align  = ucs_max(md_attr().reg_alignment, 1);
+        void *reg_address = mem->address;
+        size_t reg_length = mem->length;
+        ucs_align_ptr_range(&reg_address, &reg_length, reg_align);
+
```

Comment:
1. code duplication with test_md
2. cannot register outside the allocated buffer

Reply:
1. will think how to get rid of the code duplication
2. Let's discuss this in person?

Reply:
1. there is no duplication with test_md anymore
2. it's ok as we agreed